source/features/add-keyboard-shortcuts-to-comment-fields.js

```diff
@@ -49,10 +50,31 @@ export default function () {
 				event.preventDefault();
 			}
 		} else if (event.key === 'Escape') {
-			const cancelButton = select('.js-hide-inline-comment-form', field.form);
+			const inlineCancelButton = select('.js-hide-inline-comment-form', field.form);
 
-			if (field.value !== '' && cancelButton) {
-				cancelButton.click();
+			if (inlineCancelButton) {
+				if (field.value !== '') {
+					inlineCancelButton.click();
+				}
+			} else if (field.value === '') {
+				field.blur();
+			}
+		} else if (event.key === 'ArrowUp') {
+			if (field.id === 'new_comment_field' && field.value === '') {
+				const comments = select.all('.js-comment')
+					.filter(el => select('.author', el).textContent === getUsername());
```

Comment:
This fails because issue title edit forms also have a `.js-comment` class but don't contain `.author`. Either check if it's an issue title form or check if the `.author` element exists before accessing its `textContent`.